Description: update exclude-introspect in epydoc.config to match the new module
 locations.
Origin: Sebastian Ramacher <s.ramacher@gmx.at>
Last-Update: 2011-04-09

--- python-crypto-2.3.orig/Doc/epydoc-config
+++ python-crypto-2.3/Doc/epydoc-config
@@ -15,4 +15,4 @@ link: <a href="http://www.pycrypto.org/"
 
 # The documentation is usually built on a Linux machine; nt.py tries to
 # import the winrandom module.
-exclude-introspect: ^Crypto\.Util\.osentropy\.nt$
+exclude-introspect: ^Crypto\.Random\.OSRNG\.nt|Crypto\.Util\.winrandom$
